cluster-bootstrap/src/main/resources/reference.conf

```diff
@@ -0,0 +1,78 @@
+######################################################
+# Akka Cluster Bootstrap Config                      #
+######################################################
+
+akka.management {
+
+  # registers bootstrap routes to be included in akka-management's http endpoint
+  http.route-providers += "akka.management.cluster.bootstrap.ClusterBootstrap$"
+
+  cluster.bootstrap {
+
+    # Configuration for the first phase of bootstraping, during which contact points are discovered
+    # using the configured service discovery mechanism (e.g. DNS records).
+    contact-point-discovery {
+
+      # Define this name to be looked up in service discovery for "neighboring" nodes
+      # If undefined, the name will be extracted from the ActorSystem name
+      service-name = null
```

Comment:
I could not find the <extract> you refer to? What would it do?
The setting here is the way of documenting but not setting the value. 
The setting it to null does not actually "set it to null", since `.hasPath` on `service-name` still returns false here. So it's only a way of documenting a value that is not set

Reply:
Here i'll set it to "" though the effective name I do believe should remain `= null`

Reply:
This is the one I was thinking about: https://github.com/akka/akka/blob/master/akka-remote/src/main/resources/reference.conf#L737

Reply:
Ok i see, will do